### ISSUE TITLE
release 2.5.6 master list

### ISSUE THREAD

**Issue by alexshepard** [inaturalist/INaturalistIOS#181]
From Carrie:
 - [x] Choose Projects unreachable when editing observation

From Ken-ichi
 - [x] Reset Password not working

From Scott
 - [x] username not showing after signup from Settings
 - [x] show better errors during signup failure
 - [x] facebook login is failing
 - [ ] google login unnecessary alert
 - [ ] accept on g+ sign in fails silently
 - [x] send device locale when making new user
 - [x] add comment bar in explore obs detail not showing
 - [x] observation IDs in explore obs detail not always showing
 - [x] leaderboard showing people with 0 obs - show with * obs in that case

From Joelle
 - [x] iNat should be iNaturalist
 - [x] "Launch Tutorial"
 - [ ] Taxa does not localize when changing Network affiliation
 - [x] When I Sign Up via the settings screen the Username is blank
 - [x] Facebook login doesn't work and I am brought to the Me screen and the bottom of the bottom toolbar is cut off
 - [x] Log In screen via the Me screen is not locked to portrait orientation
 - [x] Log In screen via the Explore tab is not locked to portrait orientation
 - [ ] After I revoked google plus access I was able to sign up with my joelle@thinkmoredesign.com google plus account, but after logging out I can't log back in with google plus. (I shouldn't have to revoke access every time I log in, should I?) _this isn't happening for Alex and it sounds like it's not for Patrick either, so I'm not sure where the issue is_
 - [ ] ~~Just noting that we are rolling this out without a great iPad solution in place... not sure if this was an issue for anyone~~ 

From Alex
 - [x] don't crash during signout

**Comment by alexshepard** [comment]
@kueda does it seem reasonable to remove all local taxa objects after the user changes their network affiliation? seems extreme, but that's the only way I can think of to meet Joelle's request that Taxa immediately localize after changing network affiliation.

**Comment by kueda** [comment]
Hm, I think that's fine. I don't feel like "having" taxa locally is something users are aware of or would miss if they go away, and presumably if they're changing their network then they're on the Internet, and taxa can be re-fetched on demand.

**Comment by alexshepard** [comment]
OK, thanks. Another question: will that be enough? Or do we really need to re-fetch the user's observations to get updated Species Guess?

**Comment by kueda** [comment]
The species guess does not get updated when you change sites, so it depends on how you're rendering things in the iOS app. The way I did it originally was to show the species_guess like we did on the website, so if that's what you're still doing, removing the local taxon records will not have any effect on the observation list, but probably will have an effect on the observation detail pages. I suppose the ideal solution at this stage to match the change on the website would be to never show the `species_guess` *unless* the `taxon_id` is blank. That sounds like more changes we need to test, though. Can we just table that for another bugfix release?
